cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -47,6 +48,11 @@ var (
 		` + cmdutil.AlphaDisclaimer)
 )
 
+var (
+	csrOnly bool
```

Comment:
I don't love that we have to do this, but I don't see any other way to plum this